### PR TITLE
[2.5.12] Update CSI documentation

### PR DESCRIPTION
**Pull request by pennyscissors**
Continuation of this other PR: https://github.com/rancher/charts/pull/1675
Issue: rancher/rancher#35102

### CODE REVIEW

File: packages/rancher-vsphere-csi/charts/README.md [hunk 1]
```diff
@@ -71,3 +71,9 @@ More information on managing Secrets using kubectl [here](https://kubernetes.io/
 ## Migration
 
 The CSI migration feature is only available for vSphere 7.0 U1.
+Follow the steps [here](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/rke-clusters/cloud-providers/vsphere/out-of-tree/vsphere-volume-migration/?#) for CSI Migration.
```
Comment (aiyengar2) [inline packages/rancher-vsphere-csi/charts/README.md:74]: hmm we are linking 2.6 docs here but 2.5 docs below. Should they be consistent?

File: packages/rancher-vsphere-csi/charts/app-readme.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ CSI allows volume plugins to be installed on Kubernetes clusters as extensions.
...
 The CSI driver for vSphere is `csi.vsphere.vmware.com`.
 
+## Support Matrix
+
+| Rancher vSphere CSI | Minimum Kubernetes Release | Maximum Kubernetes Release |
+| ------------------- | -------------------------- | -------------------------- |
+| 2.3.1               | 1.19.x                     | 1.21.x                     |
+| 2.3.0               | 1.19.x                     | 1.21.x                     |
+| 2.1.1               |                            | 1.19.x                     |
+| 2.1.000             |                            | 1.19.x                     |
```
Comment (aiyengar2) [inline packages/rancher-vsphere-csi/charts/app-readme.md:16]: should we just list 1.16.x for the blank minimums?

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-vsphere-csi/charts/README.md (hunk 1 above)
Comment (PennyScissors) [reply packages/rancher-vsphere-csi/charts/README.md:74]: good catch, updating now

File: packages/rancher-vsphere-csi/charts/app-readme.md (hunk 2 above)
Comment (aiyengar2) [reply packages/rancher-vsphere-csi/charts/app-readme.md:16]: I don’t think we should set a minimum on the chart, but we can indicate in the docs that before 1.16 is not officially supported

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**